mainline/alpine/Dockerfile

```diff
@@ -1,8 +1,8 @@
 # This file is largely copied from https://github.com/nginxinc/docker-nginx/tree/master/mainline/alpine
 
-FROM alpine:3.9
+FROM alpine:3.9.2
```

Comment:
This fixes the image at the major minor and minor-minor version of the Alpine image. This allows us to control exactly what version of Alpine we are using.